### DISCUSSION TITLE
Text encoder training behaviour?

### DISCUSSION THREAD

**Discussion by ProfessorMorbius** [TheLastBen/fast-stable-diffusion#646] [General]
![encrd - Copy](https://user-images.githubusercontent.com/119132145/204105201-2714f80c-db3b-4309-9075-d20a327fb932.png)

If I have 4000 training steps and text encoding percent at 60%, it seems to train the text encoding for 4000 steps? Shouldn't it be 2400? I've attached the following picture for clarity.

**Comment by TheLastBen** [comment]
at 2400steps, it will freeze the text_encoder

**Reply by ProfessorMorbius** [reply]
Cheers, I'm really new to this stuff.
